example/flaskr/recipe.py

```diff
@@ -0,0 +1,128 @@
+from flask import Blueprint
+from flask import flash
+from flask import g
+from flask import redirect
+from flask import render_template
+from flask import request
+from flask import url_for
+from werkzeug.exceptions import abort
+
+from flaskr.auth import login_required
+from flaskr.db import get_db
+
+bp = Blueprint("recipe", __name__, url_prefix="/recipe")
+
+@bp.route("/index")
```

Comment:
When I go to /recipe/index, I see the example recipe saved on the database.  But when I go to click either "new" or "edit" the following restriction occurs:
![recipe_index](https://user-images.githubusercontent.com/44328786/99891690-8f05d880-2c21-11eb-90ef-78181164522f.JPG)
![create_forbidden](https://user-images.githubusercontent.com/44328786/99891717-d55b3780-2c21-11eb-8f47-5761d128c0f2.JPG)

Reply:
The "/ingredient/[ingredient-name]" works for pulling up information! Using "generic-butter":
![ingredients](https://user-images.githubusercontent.com/44328786/99891830-20c21580-2c23-11eb-99db-8b1f8b0a603f.JPG)

Reply:
@CHRYSICS There is no 'post' route at the moment, so no way to add or edit a recipe. Do you want me to add these functions before merging this in?

Reply:
@elaguerta I looked at routes closer and understand now. These are marked 403 until the interfaces are created.  I would say that if we merge it as is, then we can help add in the interfaces later. (understanding now, "recipe" will eventually replace "blog" routes, which makes sense).  Since these two are parallel, it should be fine to merge :)

Reply:
I am actually working on cleaning things up but want to do it after everyone else gets there changes in/at the start of the next sprint to cut down on conflicts when we have to turn everything in tonight.